components/search/index.css

```
@@ -26,49 +24,61 @@ governing permissions and limitations under the License.
 	--spectrum-search-font-family: var(--spectrum-sans-font-family-stack);
+	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties */
 	--spectrum-search-font-weight: var(--spectrum-regular-font-weight);
 	--spectrum-search-font-style: var(--spectrum-default-font-style);
 	--spectrum-search-line-height: var(--spectrum-line-height-100);
+	--mod-textfield-placeholder-font-size: var(--spectrum-font-size-100);
 
-	--spectrum-search-color-default: var(
-		--spectrum-neutral-content-color-default
-	);
+	--spectrum-search-color-default: var(--spectrum-neutral-content-color-default);
 	--spectrum-search-color-hover: var(--spectrum-neutral-content-color-hover);
 	--spectrum-search-color-focus: var(--spectrum-neutral-content-color-focus);
-	--spectrum-search-color-focus-hover: var(
-		--spectrum-neutral-content-color-focus-hover
-	);
-	--spectrum-search-color-key-focus: var(
-		--spectrum-neutral-content-color-key-focus
-	);
+	--spectrum-search-color-focus-hover: var(--spectrum-neutral-content-color-focus-hover);
+	--spectrum-search-color-key-focus: var(--spectrum-neutral-content-color-key-focus);
 
 	/* Background and Border */
-	--spectrum-search-border-width: var(--spectrum-border-width-100);
+	--spectrum-search-border-width: var(--spectrum-border-width-200);
+	--spectrum-search-border-radius: var(--spectrum-corner-radius-full);
+
+	/* Used to reassign to Textfield mods */
+	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties */
 	--spectrum-search-background-color: var(--spectrum-gray-25);
+	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties */
+	--spectrum-search-border-color-default: var(--spectrum-gray-300);
+	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties */
+	--spectrum-search-border-color-hover: var(--spectrum-gray-400);
+	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties */
+	--spectrum-search-border-color-focus: var(--spectrum-gray-800);
+	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties */
+	--spectrum-search-border-color-focus-hover: var(--spectrum-gray-900);
+	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties */
+	--spectrum-search-border-color-key-focus: var(--spectrum-gray-900);
 
 	/* Disabled */
 	--spectrum-search-color-disabled: var(--spectrum-disabled-content-color);
-	--spectrum-search-background-color-disabled: var(
-		--spectrum-disabled-background-color
-	);
-	--spectrum-search-border-color-disabled: var(
-		--spectrum-disabled-background-color
-	);
+
+	/* Used to reassign to Textfield mods */
+	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties */
+	--spectrum-search-background-color-disabled: var(--spectrum-gray-25);
+	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties */
+	--spectrum-search-border-color-disabled: var(--spectrum-disabled-border-color);
```

```suggestion
	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties -- reassign to Textfield mods */
	--spectrum-search-background-color-disabled: var(--spectrum-gray-25);
	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties -- reassign to Textfield mods */
	--spectrum-search-border-color-disabled: var(--spectrum-disabled-border-color);
```

If you format the note like this, you won't get a warning about descriptionless disables for stylelint.